contracts/MintShop1155.sol

```diff
@@ -534,6 +531,16 @@ contract MintShop1155 is Sweepable, ReentrancyGuard {
       _newPaymentReceiver);
   }
 
+
+   /**
+    Allow the shop owner to add new items to the shop,
+
+    @param _items The array of Super1155 addresses.
+  */
+  function addItems(Super1155[] memory _items) external onlyOwner {
```

Comment:
nit: `setItems` instead of `addItems` since this is a setter and not something that adds to the items array.
```suggestion
  function setItems(Super1155[] memory _items) external onlyOwner {
```